content/authors/akshay-vadher/_index.md

```diff
@@ -5,7 +5,10 @@ image: '/authors/akshay-vadher/akshay-vadher.jpg'
 ---
 <img src="akshay-vadher.jpg" alt="Akshay Vadher profile" width="150"/>
 
-
 Getting things done.
 
-I vent my thoughts [here](https://akshayvadher.github.io/).
\ No newline at end of file
+[![GitHub](../../icons/github.svg)](https://github.com/akshayvadher)
```

Comment:
nit: these svgs don't have good visibility on dark theme, we have font awesome css on the blog, using icons from there will use the correct colors